src/api/spotify.ts

```diff
@@ -0,0 +1,55 @@
+import SpotifyWebAPI from 'spotify-web-api-node';
+
+export interface ISpotifyAlbumInfo {
```

Comment:
```
export type SpotifyAlbumInfo = {
    id: string;
    name: string;
    image: string;
    url: string;
};
```

Types are usually a better call if you aren't implementing or extending (like you might with interfaces) but it doesn;t reaaaaally matter. It also lets you drop the `I` :)

Reply:
aha! i knew there was something better than `interface` but i couldn't remember `type` ll. thanks :D